resources/js/manager.js

```diff
@@ -19,9 +19,13 @@ import MetricasComponent from './components/MetricasComponent';
 import DepartamentosComponent from './components/DepartamentosComponent';
 import ConfiguracionComponent from './components/ConfiguracionComponent';
 
+import ConvocatoriasCreateComponent from './components/convocatorias/ConvocatoriasCreateComponent';
+
 const routes = [
-    { path: '/', component: DashboardComponent },
+    { path: '/', redirect: '/dashboard' },
+    { path: '/dashboard', component: DashboardComponent },
     { path: '/convocatorias', component: ConvocatoriasComponent },
+    { path: '/convocatorias/create', component: ConvocatoriasCreateComponent },
```

Comment:
in this case we can use /convocatorias/crear and considere use `nested-routes`
https://router.vuejs.org/guide/essentials/nested-routes.html

Reply:
listo